test.sh

```diff
@@ -157,3 +157,8 @@ git diff --exit-code output.txt
 echo PHP simple
 php simple.php <kjvbible_x10.txt | python3 normalize.py >output.txt
 git diff --exit-code output.txt
+
+echo OCaml simple
+ocamlopt simple.ml -o simple-ml
```

Comment:
You might want to use `ocamlopt.opt with -O3` :)

Reply:
Good call, thanks!